src/Schema/Domain/Content/Mapper/FieldDefinition/UrlFieldDefinitionMapper.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace Ibexa\GraphQL\Schema\Domain\Content\Mapper\FieldDefinition;
+
+use EzSystems\EzPlatformGraphQL\Schema\Domain\Content\Mapper\FieldDefinition\DecoratingFieldDefinitionMapper;
+use EzSystems\EzPlatformGraphQL\Schema\Domain\Content\Mapper\FieldDefinition\FieldDefinitionMapper;
+use eZ\Publish\API\Repository\Values\ContentType\FieldDefinition;
+
+class UrlFieldDefinitionMapper extends DecoratingFieldDefinitionMapper implements FieldDefinitionMapper
+{
+    private const FIELD_TYPE_IDENTIFIER = 'ezurl';
+
+    /** @var bool */
+    private $shouldExtendUrlInputType;
```

Comment:
```suggestion
    private bool $shouldExtendUrlInputType;
```

Reply:
we can't do this because PHP is set minimum 7.3
"php": "^7.3 || ^8.0",

Reply:
I am aware but constructor has strict typing. Shouldn't we also adjust property as suggested then?

Reply:
After internal sync and checking it now, it needs to stay as-is.